### DISCUSSION TITLE
免费、无需注册的 ChatGPT 代理 https://freegpt.one/

### DISCUSSION THREAD

**Discussion by keroro520** [just-talks/tech-talks#30] [3.分享创造]
> 👨‍🎓 what is freegpt.one

> 🤖 FreeGPT.one is a website that offers free AI-generated content, such as text, images, and videos, using the GPT (Generative Pretrained Transformer) technology created by OpenAI. The website allows users to generate content for various purposes, including blog posts, social media posts, and product descriptions. The generated content is non-copyrighted and can be used for personal and commercial use without any attribution. However, it's important to note that the quality of the generated content may vary and it's recommended to proofread and edit the content before using it.

**Comment by adoyle-h** [comment]
这个我也用过，有几个缺点：
1. 无法保存上下文。过一会就登出了，刷新后就没历史了。
2. 没法区分多个会话。不同话题都在同一个会话里交流的话，容易干扰对话结果。
3. 需要翻墙访问。

所以我推荐自建方案  https://github.com/just-talks/tech-talks/discussions/23
